ixmp/backend/io.py

```diff
@@ -84,7 +88,23 @@ def s_write_excel(be, s, path, item_type):
                 empty_sets.append((name, data))
             continue
 
-        data.to_excel(writer, sheet_name=name, index=False)
+        if len(data) > max_row:
+            for i in range(1, int(np.ceil(len(data) / max_row)) + 1):
+                last_row = min(max_row * i, len(data))
+                if isinstance(data, pd.Series):
+                    part = data.loc[(i - 1) * max_row:last_row]
+                else:
+                    part = data.iloc[(i - 1) * max_row:last_row, :]
+
+                if i > 1:
+                    suffix = '({})'.format(i)
+                else:
+                    suffix = ''
```

Comment:
Wouldn't it better to always add numeric prefix if number of elements is more than 1 sheet?

Reply:
Now, the sheets will be saved as foo, foo(1), foo(2), ... . I think it's better to see the sheets starting with the name of the items (in case the sheet name is not fully expanded).